content/en/news/support/advance-notice-opencensus-deprecation/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Advance notice of Istio deprecating OpenCensus
+subtitle: Support Announcement
+description: Advance notice of Istio deprecating OpenCensus.
+publishdate: 2023-08-17
+---
+
+OpenCensus has [sunsetted](https://opentelemetry.io/blog/2023/sunsetting-opencensus) on July 31st, 2023.
+
+The OpenCensus sunsetting will impact Istio's usage of OpenCensus in tracing, logging, and metrics. Istio
+plans to use OpenTelemetry exporters with the OpenTelemetry collector to provide matching telemetry capabilities.
+
+This is an advance notice of Istio deprecating OpenCensus, which is tentatively scheduled to Oct. 10, 2024.
```

Comment:
advance -> advanced

Reply:
https://www.thesaurus.com/e/grammar/advance-notice-or-advanced-notice/: _"The one that’s used more often is advance notice. Because it’s used more often, it’s considered to be the standard form by most sources. "_

Reply:
Thanks for correcting me! Didn't know that.